src/main/java/com/redhat/cloud/custompolicies/app/model/validation/ActionValidator.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.cloud.custompolicies.app.model.validation;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+/**
+ * Do the validation if passed actions are good or not.
+ * @author hrupp
+ */
+public class ActionValidator implements ConstraintValidator<ValidActionS, String> {
+
+   List<String> validActions = new ArrayList();
+
+   public void initialize(ValidActionS constraint) {
+      validActions.add("email");
+      validActions.add("webhook");
+   }
+
+   public boolean isValid(String input, ConstraintValidatorContext context) {
+      if (input == null || input.isEmpty()) {
+         return true;
+      }
+      String[] actions = input.split(";");
+      int i = 0;
+      for (String action: actions) {
+         String a = action.toLowerCase().strip();
+         if (a.isEmpty() || validActions.contains(a)) {
+            i++;
+         }
+      }
+      return i == actions.length;
```

Comment:
I don't think we want to support multiple ways to specify an empty actions ( i.e. `";"`), the default `""` seems enough.
I'm curious if you had in mind something else.

Reply:
This is still one string that can list multiple actions. Not yet an array.
So it could be "email;webhook" or someone could supply "email ; "